src/test/java/uk/gov/pay/ledger/transaction/TransactionIntegrationTest.java

```diff
@@ -18,7 +18,7 @@
 public class TransactionIntegrationTest {
 
     @ClassRule
-    public static AppWithPostgresRule rule = new AppWithPostgresRule();
+    public static AppWithPostgresAndSqsRule rule = new AppWithPostgresAndSqsRule();
```

Comment:
have we discussed the tradeoff of this approach vs [DropwizardJUnitRunner|https://github.com/alphagov/pay-connector/blob/master/src/test/java/uk/gov/pay/connector/junit/DropwizardJUnitRunner.java] (which spins up the needed containers once for a whole test run, rather than per test class)

Reply:
Not sure. I for sure need to read on different ways to spin up app and containers for testing. 